zipline/build.zig

```diff
@@ -0,0 +1,95 @@
+const std = @import("std");
+
+pub fn build(b: *std.Build) !void {
+  // The Windows builds create a .lib file in the lib/ directory which we don't need.
+  const deleteLib = b.addRemoveDirTree(b.getInstallPath(.prefix, "lib"));
+  b.getInstallStep().dependOn(&deleteLib.step);
```

Comment:
We don't do Windows at the moment, but might in the future (https://github.com/cashapp/zipline/issues/1360). It should be easier now, anyway.